docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -303,7 +303,55 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 You can see that the Container was terminated because of `reason:OOM Killed`,
 where `OOM` stands for Out Of Memory.
 
-## Opaque integer resources (Alpha feature)
+## Local ephemeral storage (alpha feature)
+
+Kubernetes version 1.8 introduces a resource for local ephemeral storage. The
+management of local ephemeral storage is very similar to memory resources.
+
+**Note:** Local ephemeral storage is an alpha feature in version 1.8.
+{: .note}
+
+Each Container of a Pod can specify one or more of the following:
+
+* `spec.containers[].resources.limits.ephemeral-storage`
+* `spec.containers[].resources.requests.ephemeral-storage`
+
+Limits and requests for `ephemeral-storage` are measured in bytes. You can express storage as
+a plain integer or as a fixed-point integer using one of these suffixes:
+E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi,
+Mi, Ki. For example, the following represent roughly the same value:
+
```

Comment:
Please add a section about what `ephemeral-storage` is and how it is measured. For example, it includes overlayFS, etc.